pages/api/graphql.ts

```diff
@@ -18,7 +18,8 @@ const handler: any = nextConnect() // For session middleware. TODO: Need to defi
 const apolloServer = new ApolloServer({
   typeDefs,
   resolvers,
-  context: ({ req }) => ({ req }) // This lets GraphQL have access to sessions
+  context: ({ req }) => ({ req }), // This lets GraphQL have access to sessions
+  uploads: false
```

Comment:
This fixes discrepancy between schema generated and normal api routes graphql endpoint (which we'll be used for server side data fetching in the future). By default apolloServer accepts uploads, while schema-generated server does not .

Reply:
Can you put this in the comment in the code?

Reply:
Done.